Cargo.toml

```diff
@@ -9,39 +9,43 @@ readme = "README.md"
 repository = "https://github.com/maidsafe/routing"
 version = "0.37.0"
 
+[[example]]
+bench = false
+name = "key_value_store"
+
+[[example]]
+bench = false
+name = "ci_test"
+
 [dependencies]
 config_file_handler = "~0.11.0"
-crust = "~0.31.0"
 fake_clock = "~0.2.0"
 hex = "~0.3.1"
 itertools = "~0.7.8"
+lazy_static = "~1.0.0"
 log = "~0.4.1"
 lru_time_cache = "~0.8.0"
 maidsafe_utilities = "~0.16.0"
 num-bigint = "~0.1.43"
 quick-error = "~1.2.1"
 rand = "~0.4.2"
-resource_proof = "~0.5.0"
-rust_sodium = "~0.10.0"
+resource_proof = "~0.6.0"
+safe_crypto = "~0.4.0"
 serde = "~1.0.37"
 serde_derive = "~1.0.37"
 term = "~0.4.6"
-tiny-keccak = "~1.3.0"
+tiny-keccak = "~1.4.2"
```

Comment:
I'm wondering, is `tiny-keccak` still necessary? `safe_crypto` provides a `hash` function now, so it would be nice if we could have one less dependency here :)